code/modules/mob/living/carbon/human/species_types/monkeys.dm

```diff
@@ -3,7 +3,6 @@
 /datum/species/monkey
 	name = "Monkey"
 	id = SPECIES_MONKEY
-	bodytype = BODYTYPE_ORGANIC | BODYTYPE_MONKEY
```

Comment:
Where are monkeys getting the bodytype from, here?

Reply:
Also this seems to be going against your recent PRs of trying to move things to species, which I was a fan of. I want to move away from the carbon level because I think it's filled with shitcode and I want to kill it ASAP

Reply:
Bodytypes are handled by bodyparts in the synchronize_bodyparts() proc, no matter the species. Simply put, they are every bodytype gotten from every bodypart, cached.
As such, bodytypes is pretty much a convenience variable, and instead of putting it on species, it'd be better to put it on /mob/living/carbon.